contracts/dca/src/handlers/execute.rs

```diff
@@ -6,39 +6,54 @@ use abstract_sdk::features::AbstractResponse;
 use cosmwasm_std::{
     wasm_execute, CosmosMsg, Decimal, DepsMut, Env, MessageInfo, Response, Uint128,
 };
-use cw_asset::{Asset, AssetList};
+use cw_asset::{Asset, AssetList, AssetListUnchecked};
 
 use crate::contract::{AppResult, DCAApp};
 
 use crate::error::AppError;
-use crate::msg::{DCAExecuteMsg, ExecuteMsg, Frequency};
 use crate::state::{Config, DCAEntry, CONFIG, DCA_LIST, NEXT_ID};
-use abstract_dex_adapter::api::DexInterface;
 use abstract_sdk::AbstractSdkResult;
 use croncat_app::croncat_integration_utils::{CronCatAction, CronCatTaskRequest};
+
+#[allow(unused)]
+use crate::msg::{DCAExecuteMsg, ExecuteMsg, Frequency};
+#[allow(unused)]
+use abstract_dex_adapter::api::DexInterface;
+#[allow(unused)]
 use croncat_app::{CronCat, CronCatInterface};
 
+/// Helper to attach funds for creation or refilling a task
+#[allow(unused)]
```

Comment:
We can just do `#![allow(unused)]` on the top of this file to replace these.

Reply:
I was thinking about it as well, but maybe it can act as a hints here, thoughts?